central/notifiers/cscc/cscc.go

```diff
@@ -158,8 +158,34 @@ func (c *cscc) Close(ctx context.Context) error {
 
 // AlertNotify takes in an alert and generates the notification
 func (c *cscc) AlertNotify(ctx context.Context, alert *storage.Alert) error {
+
+	var finding *findings.Finding
+	var findingID string
+	var err error
+
+	findingID, finding, err = c.initFinding(ctx, alert)
```

Comment:
Change to `findingID, finding, err := c.initFinding(ctx, alert)`.  If you use `:=` you don't need to declare the variables above.